.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 name: UBC Formula Electric CI
 on:
   pull_request:
+    branches:
+      - master
```

Comment:
good to have this explicit

Reply:
prevents too many CIs from running on half-merges

Reply:
Actually can you change this to any PR
I like to stack my big tickets into multiple PRs where having CI on all of them is nice
If we need more github actions minutes we can get them